ios/FluentUI/Tab Bar/TabBarItemView.swift

```diff
@@ -106,43 +124,187 @@ class TabBarItemView: UIView {
         updateColors()
     }
 
+    private struct Constants {
+        static let unselectedColor: UIColor = Colors.textSecondary
+        static let spacingVertical: CGFloat = 3
+        static let spacingHorizontal: CGFloat = 8
+        static let portraitImageSize: CGFloat = 28
+        static let portraitImageWithLabelSize: CGFloat = 24
+        static let landscapeImageSize: CGFloat = 24
+        static let badgeVerticalOffset: CGFloat = -4
+        static let badgePortraitTitleVerticalOffset: CGFloat = -2
+        static let singleDigitBadgeHorizontalOffset: CGFloat = 14
+        static let multiDigitBadgeHorizontalOffset: CGFloat = 12
+        static let badgeHeight: CGFloat = 16
+        static let badgeMinWidth: CGFloat = 16
+        static let defaultBadgeMaxWidth: CGFloat = 42
+        static let badgeBorderWidth: CGFloat = 2
+        static let badgeFontSize: CGFloat = 11
+        static let badgeHorizontalPadding: CGFloat = 10
+        static let badgeCorderRadii: CGFloat = 10
+    }
+
+    private let container: UIStackView = {
+        let container = UIStackView(frame: .zero)
+        container.alignment = .center
+        container.distribution = .fill
+
+        return container
+    }()
+
+    private let imageView: UIImageView = {
+        let imageView = UIImageView(frame: .zero)
+        imageView.contentMode = .scaleAspectFit
+        imageView.tintColor = Constants.unselectedColor
+
+        return imageView
+    }()
+
+    private let titleLabel: Label = {
+        let titleLabel = Label()
+        titleLabel.lineBreakMode = .byTruncatingTail
+        titleLabel.textAlignment = .center
+        titleLabel.textColor = Constants.unselectedColor
+
+        return titleLabel
+    }()
+
+    private let badgeView: UILabel = {
```

Comment:
large content viewer in iOS 13 does it work?

Reply:
Yes it works and matches what Apple does for the built-in UITabBar.